server/lib/routes/post-metrics.js

```diff
@@ -108,7 +110,7 @@ const BODY_SCHEMA = {
   service: STRING_TYPE.regex(SERVICE_PATTERN).required(),
   startTime: TIME_TYPE.required(),
   timers: joi.object().optional(), // this is never consumed.
-  uid: HEX32_TYPE.allow('none').optional(),
+  uid: HEX32_TYPE.allow('none').required(),
```

Comment:
`uid` has been around a few months now, it's safe to promote to `required` status.